src/components/query-form/query-form.js

```diff
@@ -1,13 +1,23 @@
-import { useState } from 'react'
+import { useState, useRef } from 'react'
```

Comment:
I'd love to see us use this same hook in the auto anchoring popover too, as that's using the old style of createRef

Reply:
Ah yes.  Good call.